testing/citest/tests/kube_smoke_test.py

```diff
@@ -389,6 +389,324 @@ def make_deploy_stage(self, imageSource=None, requisiteStages=[], **kwargs):
 
     result.update(kwargs)
     return result
+  def create_statefulset_pipeline(self):
+      bindings = self.bindings
+
+      payload = self.agent.make_json_payload_from_kwargs(
+          job=[{
+              'type': 'manual',
+              'user': '[anonymous]'
+          }],
+          description='Create StatefulSet in Application ' + self.TEST_APP,
+          application=self.TEST_APP)
+      builder = kube.KubeContractBuilder(self.kube_observer)
+      (builder.new_clause_builder('StatefulSet Added', retryable_for_secs=60)
+       .get_resources(
+          'statefulsets', extra_args=[self.TEST_APP+'tstv001', '--namespace', self.TEST_NAMESPACE])
```

Comment:
It's not clear what TEST_APP+'tstv001' is. Could you give that a symbolic name (attribute)? Since it is constant you can define it in the scenario constructor. It seems you use this name in various tests. If these are intended to refer to the same resource, then reuse that symbolic name. Otherwise give them different names.

Reply:
When we give the name as per your framework , i get below error in spinnaker.

Exception ( Determine Target Server Group )
Unable to locate current_asg_dynamic in config62/default/kubsmok210350-tst-v001

And when i used the name as kubsmok210350tstv001 its delete the statefulset/daemonset successfully.
This is why we have removed '-' in between the name.  

As per your comments i am creating a symbolic name.